scopes/toolbox/regex/node-modules-exclude-packages/node-modules-exclude-packages.docs.md

```diff
@@ -0,0 +1,25 @@
+---
+labels: ['typescript', 'utils', 'packages', 'node modules']
+description: 'Exclude packages from node modules.'
+---
+
+import { nodeModulesExcludePackages } from './node-modules-exclude-packages';
+
+A function that receive an array of packages name to exclude from node modules and return a regex of it.  
```

Comment:
you should describe that the regex catch node modules dir without specific packages. 
you are missing the fact that what you actually include is the node modules dir

Reply:
I changed the description. Please take a look.